docs/ldml/tr35-keyboards.md

```diff
@@ -591,12 +598,12 @@ _Attribute:_ `value` (required)
 
 ### <a name="Element_settings" href="#Element_settings">Element: settings</a>
 
-An element used to keep track of layout specific settings. This element may or may not show up on a layout. These settings reflect the normal practice by the implementation. However, an implementation using the data may customize the behavior.
+An element used to keep track of layout-specific settings by implementations. This element may or may not show up on a layout. These settings reflect the normal practice by the implementation. However, an implementation using the data may customize the behavior.
 
 **Syntax**
 
 ```xml
-<settings [fallback="omit"] [transformFailure="omit"] [transformPartial="hide"] />
+<settings fallback="omit" />
```

Comment:
Without context here... what is `fallback="omit"` supposed to indicate?  It's here as an example, but the meaning of the example is unclear.

Reply:
Just below there: 

> Attribute: fallback="omit"
>
> The presence of this attribute means that when a modifier key combination goes unmatched, no output is produced. The default behavior (when this attribute is not present) is to fall back to the base map when the modifier key combination goes unmatched.

Reply:
Ah, I must have searched the wrong side of the source for it.